p2panda-rs/src/operation/operation_fields.rs

```diff
@@ -47,6 +47,10 @@ pub enum OperationValue {
     /// Reference to a list of document views.
     #[serde(rename = "pinned_relation_list")]
     PinnedRelationList(PinnedRelationList),
+
+    /// Reference to a document's owner key group.
+    #[serde(rename = "owner")]
+    Owner(Relation)
```

Comment:
I often daydream about using a PinnedRelation here..... It seems like quite an elegant solution to some of the complexities around members moving in and out of a key group.

Is that something we've seriously considered in recent times?

Reply:
I know it throws up different challenges itself :rofl:

Reply:
We could do this if we store the member list of a key group on the `key_group_v1` document instead of collecting the members from all `key_group_membership_v1` documents that point at a particular key group.

Would be interesting to compare the pros and cons of this approach vs the "transitive membership" approach we discussed on the chat/this monday meet.

Reply:
Yeh, I'd really like to look through pros and cons of these two newer approaches as well as the existing "limits" approach.